versions.yaml

```diff
@@ -46,7 +46,7 @@ externals:
     url: "https://github.com/flannel-io/flannel"
     version: "v0.20.2"
```

Comment:
@Amulyam24 this version will need an update as well

Reply:
Looks like the release v0.21.0 is reverted, changed it back to use use master yaml for `ppc64le`.

Reply:
Updated the version